azure-pipelines.yml

```diff
@@ -64,7 +64,7 @@ jobs:
   - job: dist_darwin
     displayName: "Dist Darwin binary"
     pool:
-      vmImage: macOS-10.13
+      vmImage: macOS-10.14
```

Comment:
My original build was failing because Azure Pipelines no longer has a `10.13` macos image:

![Screen Shot 2021-09-07 at 1 21 59 PM](https://user-images.githubusercontent.com/630449/132399645-b0b1e144-8a83-42a5-b466-8d072b8b894a.png)

According to their docs `10.14` is the oldest they support: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml